demos/src/demo.js

```diff
@@ -4,7 +4,6 @@ document.addEventListener("DOMContentLoaded", function () {
 	document.dispatchEvent(new CustomEvent('o.DOMContentLoaded'));
 });
 
-// to demonstrate :invalid pseudo-states work (without user interaction which is how they should work in prod)
```

Comment:
I wanted to avoid adding excessive data points to the form JSONs because they are already getting huge, and if they were set to a default in the templates, everything would be invalid.

Reply:
I see what you mean, it might be better to do that though as the markup will be incorrect for users in the registry for the core experience of when they hit the `html` tab?